nutanix/data_source_nutanix_static_routes.go

```diff
@@ -0,0 +1,290 @@
+package nutanix
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	v3 "github.com/terraform-providers/terraform-provider-nutanix/client/v3"
+)
+
+func dataSourceNutanixStaticRoute() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceNutanixStaticRouteRead,
+		Schema: map[string]*schema.Schema{
+			"vpc_reference_uuid": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"api_version": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"metadata": {
+				Type:     schema.TypeMap,
+				Computed: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+			"spec": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"resources": {
```

Comment:
No schema for default static route ? It has separate fields for it

Reply:
created the schema for default , local, dynamic route.